sdk/communication/communication-chat/package.json

```diff
@@ -65,7 +65,7 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/communication-common": "1.0.0-beta.1",
+    "@azure/communication-common": "1.0.0-beta.2",
```

Comment:
Was this the only reason you needed to replace the auto-generated PR?

Reply:
Yes, restoring packages failed in CI because of this.

The autorest package-version change is just sugar to fix the user-agent that's being sent.